spec/plans/results.fmf

```diff
@@ -99,8 +99,8 @@ description: |
           # String, how long did the check run.
           duration: hh:mm:ss
 
-          # String, name of the check. Corresponds to the name used in test
-          # metadata.
+          # String, name of the check. Corresponds to the name of the check
+          # specified in test metadata.
           name: dummy
```

Comment:
So here `how` should be use as well, right?

Reply:
I did change that, but then I undo the change. My reasoning was that, just like the `name` of a test, this field sounded like the name of the check to me. This field is set by the check implementation, e.g. https://github.com/teemtee/tmt/blob/main/tmt/checks/avc.py#L178, and it's not 1:1 propagation of `how` from the specification. `how` picks a plugin, the plugin then creates results and may, in theory, set `name` to whatever would be appropriate. E.g. we can have `name: dmesg.before-test` and `name: dmesg.after-test`.

Reply:
Ok, understood. So we keep `name` as is but the `how` key should be added instead?

Reply:
Hmmm, or it's not supposed to be in the results at all?